spec/support/capture_helper.rb

```diff
@@ -1,15 +1,16 @@
 module CaptureHelper
   # capture(iface, options) { 'ping -c 125 127.0.0.1 }
-  def capture(iface, options={}, &blk)
+  def capture(options={}, &blk)
+    iface   = options[:iface]   || Pcap.lookupdev
     timeout = options[:timeout] || 0
-
-    cap = PacketGen::Capture.new(iface, options)
+    filter  = options[:filter]  || false
+    opts = { iface: iface, timeout: timeout, filter: filter} 
```

Comment:
You should not filter options here. For example, parse option is filter, so spec testing parse: false will fail.
And default value for iface is Pcap.lookupdev, but all specs use 'lo' to avoid sending packets on network.